.github/workflows/ci.yml

```diff
@@ -1,332 +1,21 @@
-        with:
-          repo: sass/sass
-          path: language
-
-      - name: Run tests
-        run: npm run js-api-spec -- --sassSassRepo ../language --sassPackage ../build/npm --browser
-        working-directory: sass-spec
-        env:
-          CHROME_EXECUTABLE: chrome
-
-  dart_tests:
-    name: "Dart tests | Dart ${{ matrix.dart_channel }} | ${{ matrix.os }}"
-    runs-on: "${{ matrix.os }}"
-
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        dart_channel: [stable]
-        # TODO(nweiz): Re-enable this when
-        # https://github.com/dart-lang/sdk/issues/52121#issuecomment-1728534228
-        # is addressed.
-        # include: [{os: ubuntu-latest, dart_channel: dev}]
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/util/initialize
-        with:
-          dart-sdk: ${{ matrix.dart_channel }}
-          github-token: ${{ github.token }}
-
-      - run: dart run grinder pkg-standalone-dev
-      - name: Run tests
-        run: dart run test -x node
-
-  # Unit tests that use Node.js, defined in test/.
-  #
-  # The versions should be kept up-to-date with the latest LTS Node releases.
-  # They next need to be rotated April 2021. See
-  # https://github.com/nodejs/Release.
-  node_tests:
-    name: "Node tests | Dart ${{ matrix.dart_channel }} | Node ${{ matrix.node-version }} | ${{ matrix.os }}"
-    runs-on: "${{ matrix.os }}"
-
-    strategy:
-      fail-fast: false
-
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        dart_channel: [stable]
-        node-version: [18]
-        include:
-          # Include LTS versions on Ubuntu
-          - os: ubuntu-latest
-            dart_channel: stable
-            node-version: 16
-          - os: ubuntu-latest
-            dart_channel: stable
-            node-version: 14
-          - os: ubuntu-latest
-            dart_channel: dev
-            node-version: 18
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/util/initialize
-        with:
-          dart-sdk: ${{ matrix.dart_channel }}
-          github-token: ${{ github.token }}
-          node-version: ${{ matrix.node-version }}
-
-      - run: dart run grinder pkg-npm-dev
-      - name: Run tests
-        run: dart run test -t node -j 2
-
-  browser_tests:
-    name: "Browser Tests | Dart ${{ matrix.dart_channel }}"
-
-    strategy:
-      matrix:
-        dart_channel: [stable]
-      fail-fast: false
-
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - uses: browser-actions/setup-chrome@v1
-    - uses: ./.github/util/initialize
-      with:
-        dart-sdk: ${{ matrix.dart_channel }}
-        github-token: ${{ github.token }}
-
-    - run: dart run grinder pkg-npm-dev
-    - name: Run tests
-      run: dart run test -p chrome -j 2
-      env:
-        CHROME_EXECUTABLE: chrome
+  test:
+    uses: ./.github/workflows/test.yml
```

Comment:
I didn't realize you could put `uses:` directly within a job.

Reply:
https://docs.github.com/en/actions/using-workflows/reusing-workflows#nesting-reusable-workflows